alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/User.kt

```diff
@@ -0,0 +1,92 @@
+package com.amazon.opendistroforelasticsearch.alerting.model
+
+import org.elasticsearch.common.io.stream.StreamInput
+import org.elasticsearch.common.io.stream.StreamOutput
+import org.elasticsearch.common.io.stream.Writeable
+import org.elasticsearch.common.xcontent.ToXContent
+import org.elasticsearch.common.xcontent.ToXContentObject
+import org.elasticsearch.common.xcontent.XContentBuilder
+import org.elasticsearch.common.xcontent.XContentParser
+import org.elasticsearch.common.xcontent.XContentParserUtils.ensureExpectedToken
+import java.io.IOException
+
+data class User(
```

Comment:
I was under the impression we wanted to decouple the `roles` from the user when storing this on the Monitor?

Reply:
Having name, roles, backend_roles as user (sub) object is much clean.  Checks are simpler. `monitor.user`